src/components/KToaster/ToastManager.ts

```diff
@@ -1,87 +1,83 @@
-import KToaster from './KToaster.vue'
-import type { Ref } from 'vue'
-import { createApp, h, ref } from 'vue'
-import type { ToasterAppearance, Toast } from '@/types'
+import { createVNode, render, ref } from 'vue'
+import type { Ref, VNode } from 'vue'
+import type { Toast, ToasterAppearance, ToasterOptions } from '@/types'
 import { ToasterAppearances } from '@/types'
+import KToaster from '@/components/KToaster/KToaster.vue'
+import { v4 as uuidv4 } from 'uuid'
 
-const APPEARANCES = Object.keys(ToasterAppearances)
+const toasterContainerId = 'kongponents-toaster-container'
 
-const DEFAULTS = {
-  id: 'toaster-container',
-  timeout: 5000,
+const toasterDefaults = {
+  timeoutMilliseconds: 5000,
   appearance: ToasterAppearances.info,
-  zIndex: 10000,
 }
 
-export default class ToastManager {
-  public toasters: Ref<Toast[]>
-  public timeout: number
-  public appearance: string
-  public id: string
-  public zIndex: number
-
-  constructor({ id, timeout, appearance, zIndex } = DEFAULTS) {
-    this.toasters = ref<Toast[]>([])
+const defaultZIndex = 10000
 
-    this.timeout = timeout
-    this.appearance = appearance
-    this.id = id
-    this.zIndex = zIndex
-
-    this.mount()
-  }
+export default class ToastManager {
+  private toastersContainer: HTMLElement | null = null
+  private toaster: VNode | null = null
+  public toasts: Ref<Toast[]> = ref<Toast[]>([])
 
-  mount() {
-    // For SSR (still works on VuePress/VitePress site, but prevents failing on the build)
+  constructor(options?: ToasterOptions) {
+    // For SSR, prevents failing on the build)
     if (typeof document === 'undefined') return
```

Comment:
can you add a console.warn inside this return statement to say something to the effect of `ToastManager can only be initialized client-side`